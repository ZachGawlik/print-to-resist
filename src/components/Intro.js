import React from 'react';
import { Link } from 'react-router-dom';
import GalleryContainer from '../containers/GalleryContainer';
import '../styles/Intro.css';

const Intro = () =>
  <div className="Intro">
    <div className="Intro__information">
      <p>
        Are you a college student? Use your printing allowance to promote social
        change!
      </p>
      <p>
        Print to Resist helps activist organizations and concerned citizens
        spread key information related to the anti-Trump movement. This site
        aims to break the internet bubble to help unite the wide movement, to
        help instruct people of their rights, and to help promote effective
        methods of resistance.
      </p>
      <p>
        Want to help?{' '}
        <a className="button_link" href="#gallery">Print and post</a> or{' '}
        <Link className="button_link" to="/create">
          Add a poster worth spreading
        </Link>
      </p>
    </div>
    <GalleryContainer />
  </div>;

export default Intro;
