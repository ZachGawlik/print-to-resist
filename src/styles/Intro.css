.Intro__information {
  margin: 50px auto 200px;
  max-width: 600px;
}
