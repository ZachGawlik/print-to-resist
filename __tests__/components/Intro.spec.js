import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Intro from '../../src/components/Intro';

describe('Intro', () => {
  it('renders as expected', () => {
    const renderer = new ShallowRenderer();
    renderer.render(<Intro />);
    const result = renderer.getRenderOutput();
    expect(result).toMatchSnapshot();
  });
});
