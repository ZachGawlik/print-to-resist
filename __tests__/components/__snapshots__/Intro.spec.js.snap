// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Intro renders as expected 1`] = `
<div
  className="Intro"
>
  <div
    className="Intro__information"
  >
    <p>
      Are you a college student? Use your printing allowance to promote social change!
    </p>
    <p>
      Print to Resist helps activist organizations and concerned citizens spread key information related to the anti-Trump movement. This site aims to break the internet bubble to help unite the wide movement, to help instruct people of their rights, and to help promote effective methods of resistance.
    </p>
    <p>
      Want to help?
       
      <a
        className="button_link"
        href="#gallery"
      >
        Print and post
      </a>
       or
       
      <Link
        className="button_link"
        replace={false}
        to="/create"
      >
        Add a poster worth spreading
      </Link>
    </p>
  </div>
  <Connect(Gallery) />
</div>
`;
